README.md

```diff
@@ -51,7 +50,7 @@ Firstly, to obtain a copy of the (Ci40) Marduk platform supported OpenWrt source
 
 Install build dependencies:
 
-    $ sudo apt-get install libncurses5-dev libncursesw5-dev zlib1g-dev libssl-dev gawk subversion device-tree-compiler
+    $ sudo apt-get install libncurses5-dev libncursesw5-dev zlib1g-dev libssl-dev gawk subversion device-tree-compiler bsdmainutils
```

Comment:
rysnc needs to be installed, it seems to be required for certain packages which aren't enabled by default.

bsdmainutils and bc are mandatory just that they are already installed by default on a standard ubuntu desktop image. However on the minimal docker images they aren't.